aperturedb/Connector.py

```diff
@@ -376,7 +376,7 @@ def _connect(self):
             if self.use_ssl:
 
                 # Server is ok with SSL, we switch over SSL.
-                self.context = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
+                self.context = ssl.create_default_context()
```

Comment:
what is `create_default_context` supposed to be? I only ask because usually in security contexts its better to be explicit about what you are configuring if possible.